test/tests/functional/pbs_accumulate_resc_used.py

```diff
@@ -64,10 +64,10 @@ def setUp(self):
 
         TestFunctional.setUp(self)
         self.logger.info("len moms = %d" % (len(self.moms)))
+        usage_string = 'test requires 3 MoMs as input, ' + \
+                        '  use -p moms=<mom1>:<mom2>:<mom3>'
```

Comment:
It looks like the two lines aren't aligned properly.  I suspect pep8 will complain.  Also, I know this is part of the existing message, but I'd remove some of the blank spaces between the two quoted strings.  There is one on the top string and two on the bottom.  The combined string will have 3 spaces where it only really needs one.
Lastly, you probably want to put the setting of the usage string inside of the if statement so it is only set when it is needed.  Right now it will be set on every pass through setUp().

Reply:
Thanks. I have corrected the space and alignment issues now.